scripts/src/java/org/oppia/android/scripts/todo/IssueTodosResolvedCheck.kt

```diff
@@ -0,0 +1,106 @@
+package org.oppia.android.scripts.todo
+
+import org.oppia.android.scripts.todo.model.Todo
+import java.io.File
+
+/**
+ * Script for ensuring that all TODOs of the closed issue are resolved.
+ *
+ * Usage:
+ *   bazel run //scripts:issue_todos_resolved_check -- <path_to_directory_root>
+ *   <closed_issue_number> <github_sha>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ * - closed_issue_number: issue number of the closed issue.
+ * - github_sha: sha of the latest commit on the develop branch.
+ *
+ * Example:
+ *   bazel run //scripts:issue_todos_resolved_check -- $(pwd) 6
```

Comment:
Nit: break at the space instead of the first digit of the hash.

Reply:
Done.